src/test/java/de/tum/in/www1/artemis/service/connectors/TextSimilarityClusteringServiceTest.java

```diff
@@ -21,9 +21,9 @@
 
 public class TextSimilarityClusteringServiceTest {
 
-    private static String CLUSTERING_ENDPOINT = "http://localhost:8000/cluster";
+    private static String CLUSTERING_ENDPOINT = "http://localhost:8002/cluster";
 
-    private static String EMBEDDING_ENDPOINT = "http://localhost:8000/embed";
+    private static String EMBEDDING_ENDPOINT = "http://localhost:8001/embed";
```

Comment:
I would change those lines for an ascending order, but does not matter at all ;)